packages/plugin-defender-hardhat/src/propose-upgrade.ts

```diff
@@ -0,0 +1,37 @@
+import '@openzeppelin/hardhat-upgrades/dist/type-extensions';
+import { getChainId, ValidationOptions } from '@openzeppelin/upgrades-core';
+import { AdminClient, ProposalResponse } from 'defender-admin-client';
+import type { ContractFactory } from 'ethers';
+import { FormatTypes } from 'ethers/lib/utils';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { fromChainId } from 'defender-base-client';
```

Comment:
Can you add `defender-base-client` as a dependency? I know it's a transitive dependency from the admin client, but (I think) this could fail depending on how npm or yarn (or whatever) lays out the dependencies on the file system if we don't declare the dependency explicitly.